### DISCUSSION TITLE
How un ailias and remove function that has been created oh-my-zsh and don't play crucial role, except plugins, themes.

### DISCUSSION THREAD

**Discussion by akonitx** [ohmyzsh/ohmyzsh#11588] [Help] [answered]
I know that i can run `zstyle ':omz:directories' aliases no` but it's only deletes aliases in directories.zsh,
but here is antoher functions and aliases as example in:
```
lib/function.zsh
plugins/git/git.plugins.zsh
and so on.
```
**How for me don't use(unalias) and delete functions that created by oh-my-zsh forever?**
i want this because oh-my-zsh creates all of messy in my using CLI interfaces in terminal.

**Comment by carlosala** [comment, marked as answer]
We are working on it, there's a WIP implementation for that in https://github.com/ohmyzsh/ohmyzsh/pull/11550. Thanks for the idea!

**Reply by akonitx** [reply]
Nice, good luck!

**Reply by akonitx** [reply]
Is somewhere a more detailed documantation than [README.md](https://github.com/ohmyzsh/ohmyzsh#readme)
as example for `zstyle`?

**Reply by carlosala** [reply]
https://zsh.sourceforge.io/Doc/Release/Zsh-Modules.html#The-zsh_002fzutil-Module
